R/get_adrac.R

```diff
@@ -49,7 +49,11 @@ get_adrac <- function(db_url = get_db_url("adrac")) {
       lon = .data[["LONG"]],
       comment = .data[["REMARK"]],
       shortref = .data[["SOURCE"]]
-    ) %>% dplyr::mutate(
+    ) %>%
+    dplyr::filter(
+      is.na(method) | method != "TL"
```

Comment:
Lazy evaluation should not be used in packages. That's the reason for the note. You could replace `method` with `.data[["method"]]`. That's our usual hack for this issue.